python/lsst/ts/standardscripts/auxtel/calsys.py

```diff
@@ -148,7 +148,7 @@ def fiberspectrograph(self):
 
         Returns
         -------
-        large_file_data : ``fiberspectrograph.evt_largeFileObjectAvailable.DataType``
+        large_file_data : ``fiberspectrograph.evt_largeFileObjectAvailable.DataType``  # noqa
```

Comment:
Is this a typo?

Reply:
The `# noqa` prevents `flake8` from complaining that the line is too long.

It would probably be better to use a different data type but I didn't want to change your text.